src/cryptenroll/cryptenroll-tpm2.c

```diff
@@ -414,7 +415,10 @@ int enroll_tpm2(struct crypt_device *cd,
                 log_debug_errno(r, "PCR policy hash not yet enrolled, enrolling now.");
         else if (r < 0)
                 return r;
-        else {
+        else if (use_pin) {
+                log_debug("This PCR set is already enrolled, rotating PIN.");
```

Comment:
"rotating PIN"?

that's weird wording, no? maybe say "This PCR set is already enrolled, re-enrolling anyway to update PIN".